test/unit/gateways/mercado_pago_test.rb

```diff
@@ -429,6 +429,16 @@ def test_invalid_taxes_shape
     end
   end
 
+  def test_successful_purchase_with_payment_method_option_id
+    payment_method_option_id = '1234'
+
+    stub_comms do
+      @gateway.purchase(@amount, @credit_card, @options.merge(payment_method_option_id: payment_method_option_id))
+    end.check_request do |endpoint, data, headers|
+      assert_match("\"payment_method_option_id\":\"#{payment_method_option_id}\"", data) if endpoint =~ /payments/
```

Comment:
Not sure that I understand what the `if` statement is doing here. Could you explain your thoughts on that?

Reply:
That began as an artifact of duplicating code. It stayed from a misunderstanding of the documentation. It is probably not appropriate here.
<img width="638" alt="Screen Shot 2020-05-15 at 10 41 26 AM" src="https://user-images.githubusercontent.com/13825990/82062796-a2475280-9698-11ea-946c-565eac7bfb84.png">

Regardless, I think this entire test case would go away. There's not currently anything in the unit test that deals with processing options, and adding them may increase the scope of this pull request.

Reply:
007ab496410a32775742ba9d92258cbc61952732

I've removed this test case for now